browser/ui/views/playlist/playlist_action_icon_view.cc

```diff
@@ -27,43 +24,35 @@ PlaylistActionIconView::PlaylistActionIconView(
                          page_action_icon_delegate,
                          "PlaylistActionIconView",
                          /*ephemeral=*/false),
-      browser_(browser) {
-  SetVisible(false);
```

Comment:
I'm curious why the default visibility setting was removed.

Reply:
Doesn't make a difference in practice, as `UpdateState()` sets the right visibility.

Reply:
Actually, added this back for clarity. 👍🏻